docs/development/use-bundled-plugins.md

```diff
@@ -0,0 +1,63 @@
+
+#### Install and run a "full" version of OpenSearch via tarball
+
+_(Do this on the same machine where you're developing on OpenSearch Dashboards - these examples assume you're using [an Ubuntu dev environment](https://quip-amazon.com/umanA1aLbFsw/Setup-Ubuntu-Machine-for-OpenSearch-devtest))_
+
+1. Make/navigate to a directory where you want to store/run OpenSearch tarballs
+2. Download the latest version of the OpenSearch tarball
+  `curl https://artifacts.opensearch.org/releases/bundle/opensearch/{VERSION_NUMBER}/opensearch-{VERSION_NUMBER}-linux-x64.tar.gz --output opensearch-{VERSION_NUMBER}-linux-x64.tar.gz`
+  for example, if you want version `2.1.0`,
+  `curl https://artifacts.opensearch.org/releases/bundle/opensearch/2.1.0/opensearch-2.1.0-linux-x64.tar.gz --output opensearch-2.1.0-linux-x64.tar.gz`
+3. Unzip it: `tar -zxf opensearch-{VERSION_NUMBER}-linux-x64.tar.gz`
+4. `cd` inside and run `./opensearch-tar-install.sh ` (this is the only step necessary to subsequently restart the service)
+
+_(The above is a slightly streamlined version of [the official tarball installation documentation](https://opensearch.org/docs/latest/opensearch/install/tar/))_
+
+### 2. Install the OpenSearch Dashboards security plugin
+
+Within the `plugins` directory of your local `OpenSearch-Dashboards` repo, clone the latest version of the security dashboards plugin: `git clone git@github.com:opensearch-project/security-dashboards-plugin.git`
+
+### 3. Update the version strings to pass the version match test
+
+In `plugins/security-dashboards-plugin/opensearch_dashboards.json` update the `opensearchDashboardsVersion` and  `version` strings to match the current version of OpenSearch Dashboards as defined in `package.json`.
+
+### 4. Bootstrap
+
+Run `yarn osd bootstrap` to build the newly installed plugin.
+
+### 5. Update your config with minimal settings
+
+The security plugin requires some config values to be set to start properly. Add the following configurations to `config/opensearch_dashboards.yml`:
+
+```yml
+csp.warnLegacyBrowsers: false
+opensearch.ignoreVersionMismatch: true
+
+opensearch.hosts: [https://localhost:9200]
+opensearch.ssl.verificationMode: none
+opensearch.username: kibanaserver
+opensearch.password: kibanaserver
+opensearch.requestHeadersWhitelist: [authorization, securitytenant]
+
+opensearch_security.multitenancy.enabled: true
+opensearch_security.multitenancy.tenants.preferred: [Private, Global]
+opensearch_security.readonly_mode.roles: [kibana_read_only]
+# Use this setting if you are running opensearch-dashboards without https
+opensearch_security.cookie.secure: false
+```
```

Comment:
Is there an authoritative source for this config, ideally in the security plugin repo that we can point them to if this changes? that way we can keep this as an example but call out that it is not always going to be accurate. Right now it looks like this is the only copy of the config changes that i can find, which should not be the case.